### ISSUE TITLE
When battery saving mode is enabled tooltip boucing rapidly

### ISSUE THREAD

**Issue by nishantpardamwar** [sephiroth74/android-target-tooltip#76]
When i enable battery saving mode, tooltip bouncing animation starts to malfunction, as soon as battery saving mode is disabled it gets normal, is there any fix for this ?

**Comment by laxman-spidey** [comment]
By disabling floatingAnimation you can stop the rapid bouncing, we'll missing bouncing effect entirely. So, as a workaround I'm checking if battery saving mode is enabled and enabling and disabling the effect. Below code may help.

`boolean batterySaverEnabled = false;
            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
                PowerManager powerManager = (PowerManager) context.getSystemService(Context.POWER_SERVICE);
                if (powerManager.isPowerSaveMode()) {
                    batterySaverEnabled = true;
                }
            }`

`if (!batterySaverEnabled) {
                builder.floatingAnimation(Tooltip.AnimationBuilder.DEFAULT);
            }`

**Comment by nishantpardamwar** [comment]
Thanks for the answer, but i still if power saver is enabled when the tooltips were already showing then this wont work right?

**Comment by laxman-spidey** [comment]
Yeah. good point.. we may have to catch event from PowerManager (if there is any API available. I didn't find any yet 😛) and re-create the Tooltip. 
In my case Tooltip stays only for 3-5 seconds, this kind of case could be rare. I'm waiting for someone to fix this in the Library itself.

**Comment by nishantpardamwar** [comment]
@laxman-spidey you can listen to power saver event with action ACTION_POWER_SAVE_MODE_CHANGED, its there in the documentation itself.

> You can monitor for changes to this state with ACTION_POWER_SAVE_MODE_CHANGED.

**Comment by TuNguyen90Vn** [comment]
Hi everyone. I am using this library on my project. I have a question. How to hidden status bar when show tooltip??

Many thanks!
